test/functional/webview-browser-tester-e2e-specs.js

```diff
@@ -0,0 +1,53 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../..';
+import path from 'path';
+import { sleep } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const AVD_ANDROID_24_WITHOUT_GMS = "Nexus_5_API_24";
+const capabilities = {
+  "browserName": "chromium-webview",
+  "avd": AVD_ANDROID_24_WITHOUT_GMS,
+  "platformName": "Android",
+  "platformVersion": "7.0",
+  "deviceName": "Android Emulator",
+  "chromedriverExecutable": path.join(process.cwd(), "chromedriver")
+};
+
+describe('Android 7 Webview Browser tester', function () {
+  let driver, test = this;
+  before(() => {
+    if (process.env.REAL_DEVICE) {
+      test.pending = true;
```

Comment:
Yes, it's skipped but set as _pending_. If we use `it.skip` it shows like it's "passing"

<img width="801" alt="screen shot 2017-03-20 at 1 31 48 pm" src="https://cloud.githubusercontent.com/assets/329957/24110228/a61a7260-0d71-11e7-92f9-b7307be4b40b.png">

what do you guys prefer ? I took a screenshot of both cases.